src/background/modules/ContextMenu.js

```diff
@@ -12,6 +11,38 @@ const PREVIEW_STRING_CUT_LENGTH = 100; // a setting that may improve performance
 let lastCachedUnicodeFontSettings = null;
 let menuIsShown = false;
 
+let pasteSymbol = null;
+
+/**
+ * Create notification.
+ *
+ * @param {string} title
+ * @param {string} message
+ * @returns {void}
+ */
+function notification(title, message) {
+    console.log(title, message);
```

Comment:
```suggestion
    console.info("Showing notification:", title, message);
```

If this is intended to be a log, otherwise remove it.

Reply:
Yes, that was intended.